types/parser/type_parser.go

```diff
@@ -3,6 +3,7 @@
 package parser
```

Comment:
I generally think we're just creating more debt by maintaining multiple parsers in the project rather than just using antlr everywhere.

Reply:
We will fix it in follow up PR to rely on antlr

Reply:
Is there an antlr implementation for Go that we can use which is easy to utilize and plug into the types?

Reply:
Antlr has generators for most languages, including go. Have used it extensively elsewhere and much more reliable/easy to maintain than writing parsers in multiple languages.